Package.swift

```diff
@@ -0,0 +1,32 @@
+// swift-tools-version:5.3
+import PackageDescription
+
+let package = Package(
+    name: "SelfSignedCert",
+    platforms: [
+        .macOS(.v10_14), .iOS(.v13), .tvOS(.v13)
+    ],
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "SelfSignedCert",
+            targets: ["SelfSignedCert"]
+        )
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        .package(url: "https://github.com/iosdevzone/IDZSwiftCommonCrypto", from: "0.13.1"),
+        .package(name: "SecurityExtensions", url: "https://github.com/skogland/swift-SecurityExtensions", from: "5.0.0"),
+        .package(name: "SwiftBytes", url: "https://github.com/skogland/swift-bytes.git", from: "0.8.0")
```

Comment:
The swift-bytes repo needs a version tag on the release with SPM-support to be included. This points to my fork with a version tag. Update the URL when the master is tagged. See https://github.com/dapperstout/swift-bytes/issues/5